# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/powerpc-utils/makefile.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-08-10 22:50:32.715450936 +0200
+++ ./Makefile	2004-08-10 22:52:37.634460360 +0200
@@ -27,7 +27,7 @@
 INSTALL	= /usr/bin/install -c
 SOUND_INC = -I.
 
-PROGS	= clock mousemode nvsetenv trackpad backlight nvsetvol fblevel fnset
+PROGS	= clock mousemode nvsetenv trackpad backlight nvsetvol fblevel fnset fdeject nvvideo sndvolmix
 SCRIPTS	= macos
 
 GENERATED_MAN = clock.8 macos.8 nvsetenv.8 sndvolmix.8 fdeject.1 \
@@ -130,23 +130,16 @@
 		chmod 600 $(DESTDIR)$(devsdir)/sndstat; \
 	fi
 
-install:	all man installdirs
+install:	all installdirs
 	$(INSTALL) -m 4511  clock $(DESTDIR)$(sbindir) 
 	$(INSTALL) -m 755   nvsetenv nvvideo $(DESTDIR)$(sbindir)
-	$(INSTALL) -m 755   mousemode sndvolmix vmode nvsetvol $(DESTDIR)$(usbindir)
+	$(INSTALL) -m 755   mousemode sndvolmix nvsetvol $(DESTDIR)$(usbindir)
 	$(INSTALL) -m 755   fdeject $(DESTDIR)/$(ubindir)
 	$(INSTALL) -m 755   $(SCRIPTS) $(DESTDIR)$(sbindir)
 
-	$(INSTALL) -m 644   fdeject.man	$(DESTDIR)$(man1dir)/fdeject.1
-	$(INSTALL) -m 644   clock.man	$(DESTDIR)$(man8dir)/clock.8
-	$(INSTALL) -m 644   mousemode.man	$(DESTDIR)$(man8dir)/mousemode.8
-	$(INSTALL) -m 644   nvsetenv.man	$(DESTDIR)$(man8dir)/nvsetenv.8
-	$(INSTALL) -m 644   nvvideo.man	$(DESTDIR)$(man8dir)/nvvideo.8
-	$(INSTALL) -m 644   sndvolmix.man	$(DESTDIR)$(man8dir)/sndvolmix.8
-	$(INSTALL) -m 644   vmode.man	$(DESTDIR)$(man8dir)/vmode.8
-	$(INSTALL) -m 644   macos.man	$(DESTDIR)$(man8dir)/macos.8
 	$(INSTALL) -m 644   trackpad.8	$(DESTDIR)$(man8dir)/trackpad.8
 	$(INSTALL) -m 644   nvsetvol.8	$(DESTDIR)$(man8dir)/nvsetvol.8
+	$(INSTALL) -m 644   fblevel.8	$(DESTDIR)$(man8dir)/fblevel.8
 
 cleanobjs:
 	$(RM) *.o *.bak *~ *.man $(GENERATED_MAN)
